.cursor/rules/$c.date.md

```
@@ -0,0 +1,144 @@
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)